types/firefox-webext-browser/firefox-webext-browser-tests.ts

```diff
@@ -5,7 +5,7 @@ browser.nonexistentNS.unknownMethod(); // $ExpectError
 browser.runtime.getManifest(); // $ExpectType WebExtensionManifest
 browser.test; // $ExpectError
 browser.manifest; // $ExpectError
-browser._manifest; // $ExpectType typeof _manifest
+// browser._manifest; // $ExpectType typeof _manifest
```

Comment:
You probably just need to replace `typeof _manifest` with whatever the type actually resolves to now, since it doesn't remain a type query (it is instead a union of literals, probably).